integration_tests/src/test/kotlin/utils/LoadTestSuites.kt

```diff
@@ -0,0 +1,8 @@
+package utils
+
+import com.fasterxml.jackson.dataformat.xml.XmlMapper
+import com.fasterxml.jackson.module.kotlin.KotlinModule
+import utils.testResults.TestSuites
+import java.io.File
+
+fun File.loadAsTestSuite(): TestSuites = XmlMapper().registerModule(KotlinModule()).readValue(this, TestSuites::class.java)
```

Comment:
Again with long lines and detekt failing. Do you have githooks installed?

Reply:
Yes, I have installed it.
Strange. To be sure I run detekt and check and both pass.